### ISSUE TITLE
a11y: Color contrast blue on white

### ISSUE THREAD

**Issue by sirrah-tam** [fusioncms/fusioncms#307]
### Describe the issue
Link text does not meet minimum AA contrast ratio (4.5:1) for text that is 18px or less.
Foreground: #45bbcb
Background: #ffffff
Ratio:  2.28:1

### To reproduce
Check color contrast using a color contrast analyzer tool 
Ex. https://webaim.org/resources/contrastchecker/

### Expected behavior
Text that is 18pt (24px) or less and not bold should have a contrast ratio of 4.5:1.
If text is larger than 18pt (24px) or is bold, minimum contrast ratio is 3:1.
Reference: https://www.w3.org/WAI/WCAG21/quickref/#contrast-minimum

### Screenshots
Example of link color
<img width="772" alt="color contrast blue on white" src="https://user-images.githubusercontent.com/6874453/72868392-c240a980-3c96-11ea-9a73-ddc8502fb48d.png">

### Version and browser
Version: 6.0.0-beta.2
Browser and version: Chrome v79.0.3945.117

### Bug report checklist
- [X] I have filled out as much information as I can
- [X] I have detailed how the issue can be reproduced
- [X] I have [searched for existing issues](https://github.com/fusioncms/fusioncms/issues) and to the best of my knowledge this is not a duplicate

**Comment by sirrah-tam** [comment]
The app that I use to test these things, [ANDI](https://www.ssa.gov/accessibility/andi/help/install.html), showed a recommended color of #0c8292, I probs should/could have mentioned that in the original issue.

I wasn't sure how best to approach recommendations that infringe on styles/color preference, so just wanted to point it out as an issue.

**Comment by sirrah-tam** [comment]
I lied, the color contrast ratio text size is 18pt which would be 24px, updating the issue accordingly.

**Comment by kaidesu** [comment]
![lied](https://media.giphy.com/media/bhijWb5DJt7tm/giphy.gif)

**Comment by thekanbo** [comment]
Proposing #0064e2 for link color

**Comment by sirrah-tam** [comment]
That checks out, contrast ratio of 5.4:1 👍
